### ISSUE TITLE
edge maps are computed using pretrained RCF

### ISSUE THREAD

**Issue by zachzhang07** [rajeevyasarla/3SD#2]
edge maps are computed using pretrained RCF : https://github.com/yun-liu/rcf

_Originally posted by @rajeevyasarla in https://github.com/rajeevyasarla/3SD/issues/1#issuecomment-1071526527_

Sorry to open a new issue.
Does the 'pretrained RCF' mean the model trained on the BSDS500 which is labelled by humans?

**Comment by rajeevyasarla** [comment]
Yes, we followed the same protocol for using edges used by Zhang et al. (https://arxiv.org/pdf/2003.07685.pdf). pre-trained RCF is trained on BSDS500 for computing edges.

**Comment by zachzhang07** [comment]
No offense, but this may be a little different from "no labels needed". However, the results of 'PCM' in the ablation study also demonstrate that good results can still be achieved without RCF. 
Thank you for your work.

**Comment by rajeevyasarla** [comment]
Thank you for the suggestion. even though CAM map produces good results (refer PCM in ablation study), the CAM maps don't have a proper boundary so we needed to incorporate a gated edge to generate pseudo-GT.
Computing edges RCF gave us good results, but RCF can always be replaced with a sobel filter or canny edge detector. 

we conducted an experiment with Sobel filter results are below. As Sobel filter is sensitive to noise we used RCF. 

experiment, where pseudo GT is computed using CAM and edge obtained using sobel filter
```
----------------------------------------------------
Test set    |   F         S         E       | 
----------------------------------------------------
DUTS       |  0.748  0.810  0.850 |
----------------------------------------------------
```
